CsprSdkStandardTestsNet/Test/Steps/Blocks.cs

```diff
@@ -0,0 +1,139 @@
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+using Casper.Network.SDK;
+using Casper.Network.SDK.Types;
+using CsprSdkStandardTestsNet.Test.Utils;
+using NUnit.Framework;
+using TechTalk.SpecFlow;
+using static System.Console;
+
+namespace CsprSdkStandardTestsNet.Test.Steps;
+
+[Binding]
+public class Blocks
+{
```

Comment:
I [swear] hate this brace format I know most C# devs use it. Can we use K&R style?